keps/sig-cloud-provider/20190125-removing-in-tree-providers.md

```diff
@@ -76,27 +76,29 @@ document, and merits discussion in a broader forum and input from SIG-Architectu
 considered to be safe to be vendored outside of the K8s/K8s repo and which should eventually be fully separated from
 the K8s/K8s repo. Contents of Staging are prevented from depending on code in K8s/K8s which are not in Staging.
 Controlled by [publishing kubernetes-rules-configmap](https://github.com/kubernetes/publishing-bot/blob/master/configs/kubernetes-rules-configmap.yaml)
+- **In-tree**: code that lives in the core Kubernetes repository `k8s.io/kubernetes`.
```

Comment:
It might make more sense if we specify it as 'https://github.com/kubernetes/kubernetes'

Reply:
done